.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,35 @@
+# Description
+
+Please include a summary of the change or the fix you've made. Please also include relevant motivation and context. List any dependencies that are required for this change.
+
+Fixes # (issue)
+
+## Type of change
+
+Please select atleast one
+
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
+- [ ] This change requires a documentation update
+
+## Testing
+How was this change tested?
```

Comment:
We could add a checklist to remind folks that we use unit, snapshot, and ui tests.

Reply:
We don't do snapshot tests for the Android libraries.

Reply:
@zach-klippenstein are there any tests that are Android only other than the ones Justin mentioned?

Reply:
Nope just the two.